stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -1087,7 +1087,6 @@ public struct ${Self}<Element>
 %   end
 
   public typealias Index = AnyIndex
-  public typealias IndexDistance = Int64
```

Comment:
Looks like we failed to update `count`:

```
/Users/dgregor/Projects/swift/swift/stdlib/public/core/ExistentialCollection.swift.gyb:1209:14: note: candidate has non-matching type 'Int64' [with Element = AnyRandomAccessCollection.Element, SubSequence = AnyRandomAccessCollection.SubSequence]
  public var count: Int64 {
             ^
```

I'm updating this PR and will fix.

Reply:
Once I get done on another PR factoring conformances into extensions, hopefully the near-miss warnings will actually become fewer and useful and catch stuff like this...

Reply:
Oh, that PR will conflict horribly with this one btw

Reply:
Ugh. This one might be annoying to rebase if there are major changes in master. Oh well!

Reply:
It wasn't so bad.